scripts/process-podspecs.sh

```diff
@@ -28,7 +28,7 @@ fi
 cd "$SPEC_REPO_DIR"
 SPEC_REPOS="$(git remote get-url origin),https://github.com/CocoaPods/Specs.git"
 
-POD_LINT_OPT="--verbose --allow-warnings --fail-fast --private --swift-version=3.0 --sources=$SPEC_REPOS"
+POD_LINT_OPT=(--verbose --allow-warnings --fail-fast --private --swift-version=3.0 --sources="$SPEC_REPOS")
```

Comment:
hmm this one is now triggering warning:

(warning) The = here is literal. To assign by index, use ( [index]=value ) with no spaces. To keep as literal, quote it.
See https://github.com/koalaman/shellcheck/wiki/SC2191

Perhaps it should be:

```
POD_LINT_OPT=(--verbose --allow-warnings --fail-fast --private "--swift-version=3.0" "--sources=$SPEC_REPOS")
```
?

Reply:
pushed a fixed version, let's see. Weird it's not showing here (shellcheck 0.7.2)